bazel/BUILD

```diff
@@ -7,9 +7,11 @@ load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")
 licenses(["notice"])  # Apache v2.0
 
 package(
+    default_applicable_licenses = ["//:license"],
     default_visibility = [
         "//:__subpackages__",
     ],
+    licenses = ["notice"],  # Apache 2.0
```

Comment:
Do I have to repeat the notice license here, or is it sufficient to have the `default_applicable_licenses` reference @aiuto ?